lib/streamlit/elements/media_proto.py

```diff
@@ -161,7 +162,7 @@ def _marshall_av_media(coordinates, proto, data, mimetype):
     # Assume bytes; try methods until we run out.
     if isinstance(data, bytes):
         pass
-    elif isinstance(data, io.BytesIO):
+    elif isinstance(data, io.BytesIO) or isinstance(data, UploadedFile):
```

Comment:
Since `UploadedFile` is an subclass of `io.BytesIO`, you don't need to add this.

```
>>> class Foo:
...   pass
... 
>>> class Bar(Foo):
...   pass
... 
>>> b = Bar()
>>> isinstance(b, Bar)
True
>>> isinstance(b, Foo)
True
>>> 
```

It could benefit from a comment if you remove it.